h2o-docs/src/product/performance-and-prediction.rst

```diff
@@ -2004,6 +2004,108 @@ Examples:
         # build the standardized coefficient magnitudes plot:
         glm.std_coef_plot()
 
+
+Gains/Lift 
+''''''''''
+
+Gains/Lift evaluates the prediction ability of a binary classification model. The chart is computed using the prediction probability and the true response labels. The accuracy of the classification model for a random sample is evaluated according to the results when the model is and is not used. The Gains/Lift chart shows the effectiveness of the current model(s) compared to a baseline, allowing users to quickly identify the most useful model. 
+
+To compute Gains/Lift, H2O applies the model to the original dataset to find the response probability. The data is divided into groups by quantile thresholds of the response probability. The default number of groups is 16; if there are fewer than sixteen unique probability values, then the number of groups is reduced to the number of unique quantile thresholds. 
+
+**An example**: a response model predicts who will respond to a marketing campaign. If you have a response model, you can make more detailed predictions. You use the response model to assign a score to all 100000 customers and predict the results of contacting only the top 10000 customers, the top 20000 customers, and so on. You do this by:
+
+- taking the dataset, sending it through your model, and obtaining a list of predicted output which is the probability of positive response;
+- sorting your dataset according to the output of your model which is the probability of positive response (this probability can also be called the **score**)
+    
```

Comment:
Please change 

sorting your dataset according to the output of your model which is the probability of positive response (this probability can also be called the **score**)

to

sorting your dataset according to the output of your model which is the probability of positive response (this probability can also be called the **score**) from highest to lowest;